web/src/popover_menus.js

```diff
@@ -256,6 +273,140 @@ export function initialize() {
         },
     });
 
+    tippy_no_propagation("#stream_filters .topic-sidebar-menu-icon", {
+        placement: "right",
```

Comment:
In #24795, some default options were added for left sidebar popovers. You might want to think about using them - it'll make your code more efficient and help us maintain consistency throughout the project.

Reply:
Have migrated it to use the default options.